config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root "users#index"
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get "/users", to: "users#index"
+  get "/users/:id", to: "users#show"
+  get "/users/:user_id/posts", to: "posts#index"
+  get "/users/:user_id/posts/:id", to: "posts#show"
```

Comment:
- [ ] [Optional] It would be better professionally if you add your routes like that:
```ruby
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
end

```